lib/identity_cache/configuration_dsl.rb

```diff
@@ -279,7 +279,7 @@ def attribute_dynamic_fetcher(attribute, fields, values) #:nodoc:
       end
 
       def add_parent_expiry_hook(options)
-        child_class = options[:association_class]
+        child_class = options[:association_class].klass
```

Comment:
Adding `require "associatedmodel"` isn't as ugly as having to explicitly specify in the associated model that it needs to expire cached associations on the parent model.